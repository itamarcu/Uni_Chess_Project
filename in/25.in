difficulty 4
start
move <2,E> to <4,E>
move <1,D> to <5,H>
move <1,F> to <5,B>
move <2,G> to <3,G>
move <3,G> to <4,G>
move <1,G> to <3,F>
move <2,D> to <3,D>
move <1,E> to <2,E>
move <2,A> to <4,A>
move <1,A> to <3,A>
move <3,A> to <3,C>
move <3,F> to <5,E>
move <3,C> to <7,C>
move <7,C> to <8,C>
move <8,C> to <8,D>
quit
