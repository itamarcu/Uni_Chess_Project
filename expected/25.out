 Chess
-------
Specify game settings or type 'start' to begin a game with the current settings:
Difficulty level is set to hard
Starting game...
8| R N B Q K B N R |
7| M M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ _ |
4| _ _ _ _ _ _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m m m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move pawn at <7,A> to <5,A>
8| R N B Q K B N R |
7| _ M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| M _ _ _ _ _ _ _ |
4| _ _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b q k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move pawn at <5,A> to <4,A>
8| R N B Q K B N R |
7| _ M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ _ _ _ _ _ _ q |
4| M _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b _ k b n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <8,A> to <7,A>
8| _ N B Q K B N R |
7| R M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ _ _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m m m |
1| r n b _ k _ n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <7,A> to <8,A>
8| R N B Q K B N R |
7| _ M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ _ _ |
3| _ _ _ _ _ _ m _ |
2| m m m m _ m _ m |
1| r n b _ k _ n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <8,A> to <7,A>
8| _ N B Q K B N R |
7| R M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ m _ |
3| _ _ _ _ _ _ _ _ |
2| m m m m _ m _ m |
1| r n b _ k _ n r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <7,A> to <8,A>
8| R N B Q K B N R |
7| _ M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ m _ |
3| _ _ _ _ _ n _ _ |
2| m m m m _ m _ m |
1| r n b _ k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <8,A> to <7,A>
8| _ N B Q K B N R |
7| R M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ m _ |
3| _ _ _ m _ n _ _ |
2| m m m _ _ m _ m |
1| r n b _ k _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Computer: move rook at <7,A> to <8,A>
8| R N B Q K B N R |
7| _ M M M M M M M |
6| _ _ _ _ _ _ _ _ |
5| _ b _ _ _ _ _ q |
4| M _ _ _ m _ m _ |
3| _ _ _ m _ n _ _ |
2| m m m _ k m _ m |
1| r n b _ _ _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
Illegal move
Enter your move (white player):
Illegal move
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
Computer: move pawn at <7,G> to <6,G>
8| R N B Q K B N R |
7| _ M M M M M _ M |
6| _ _ _ _ _ _ M _ |
5| _ b _ _ n _ _ q |
4| M _ _ _ m _ m _ |
3| _ _ _ m _ _ _ _ |
2| m m m _ k m _ m |
1| r n b _ _ _ _ r |
  -----------------
   A B C D E F G H
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
The specified position does not contain your piece
Enter your move (white player):
Exiting...
